src/Driver.cpp

```diff
@@ -311,10 +317,67 @@ int Driver::checkNotificationCommandMode(string const& buffer) const
 {
     if (buffer.size() < 4)
         return 0;
+    // All 4 letters notification
     else
```

Comment:
Get rid of the "else" block to avoid long indentations. Most of the test cases return, so it's clean and safe to do so.